apps/srt-live-transmit.cpp

```diff
@@ -771,12 +779,46 @@ int main(int argc, char** argv)
                     break;
                 }
 
+
+                bool srcReady = false;
+
+                if (src.get() && src->IsOpen())
+                {
+                    if (srtrfdslen > 0)
+                    {
+                        SRTSOCKET sock = src->GetSRTSocket();
+                        if (sock != SRT_INVALID_SOCK)
+                        {
+                            for (int n = 0; n < srtrfdslen; n ++)
+                                if (sock == srtrwfds[n])
+                                {
+                                    srcReady = true;
+                                    break;
+                                }
```

Comment:
More readable and error-prone.
```suggestion
                            for (int n = 0; n < srtrfdslen; n ++)
                            {
                                if (sock == srtrwfds[n])
                                {
                                    srcReady = true;
                                    break;
                                }
                            }
```